### DISCUSSION TITLE
Chrome doesn't display social links

### DISCUSSION THREAD

**Discussion by ddd-workshop** [mmistakes/minimal-mistakes#203] [Q&A] [answered]
Author's twitter, facebook, email, etc. are not displayed in Chrome.
In Firefox everything works as it should.

**Comment by mmistakes** [comment, marked as answer]
Chances are you have an ad blocking plugin installed. They often remove social links and you need to whitelist them or change some settings.

**Comment by ddd-workshop** [comment]
Yeap, that was AdBlock.
Any clues why is it blocking those links by default?

**Comment by mmistakes** [comment]
Not sure. All I know is a lot of ad blockers are overly aggressive and
remove anything faintly associated with ad networks or social networking.
They're trying to block privacy intrusive social scripts (which the theme
doesn't use) and by consequence the Fint Awesome icons and links get
stripped.

Check the closed issues. Pretty sure this has come up a bunch before.

**Comment by Amorymeltzer** [comment]
Yup - https://github.com/mmistakes/minimal-mistakes/issues/78#issuecomment-54005612

**Comment by claudiuconstantin** [comment]
I am encountering the same issue in MinimalMistakes v `4.16.4`, but this time is in Firefox x64 `v67.0.4` + uBlock Origin `1.20.0`. Everything works fine on Chrome x64 `v75.0.3770.100` and the same uBlock Origin version. The problem can be reproduced on the [official demo site](https://mmistakes.github.io/minimal-mistakes/about/).

![image](https://user-images.githubusercontent.com/1537573/60755894-fd330680-9ffe-11e9-8275-46a8d2cd4e23.png)
